### PR TITLE
Fixes #13725: Add zypper pattern doc to the package state doc

### PR DESCRIPTION
**Pull request by Fdall**
https://www.rudder-project.org/redmine/issues/13725

### CODE REVIEW

File: tree/30_generic_methods/package_state.cf [hunk 1]
```diff
@@ -86,6 +86,19 @@
...
+# ##### zypper_pattern
+#
+# This package provider uses zypper with the "-t pattern" option to manage sles patterns or
+# meta-packages on the system.
+# Since a sles pattern can be named differently than the rpm package name providing it, please
+# always use the explicit pattern name (as listed in the output of "zypper patterns")
+# when using this provider.
+#
+# Note: When installing a pattern from a local rpm file, Rudder assumes that the pattern is built
+# following the official sles documentation:
```
[the remark this answers is not in the document]
Comment (amousset) [reply tree/30_generic_methods/package_state.cf]:
```
[officiel zypper documentation](https://doc.opensuse.org/projects/libzypp/HEAD/zypp-pattern-packages.html)
```
(generic methods docs are formatted in Markdown)

File: tree/30_generic_methods/package_state.cf [hunk 2]
```diff
@@ -86,6 +86,19 @@
...
+#  <https://doc.opensuse.org/projects/libzypp/HEAD/zypp-pattern-packages.html>.
+# Older implementations of sles patterns may not be supported by this module.
```
Comment (amousset) [inline tree/30_generic_methods/package_state.cf]: sles -> zypper

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Pull request merged**